text/0000-ternary.md

```diff
@@ -0,0 +1,238 @@
++ Feature name: (fill me in with a unique ident, `my_awesome_feature`)
++ Start date: (fill me in with today's date, YYYY-MM-DD)
++ RFC PR: [iotaledger/bee-rfcs#0000](https://github.com/iotaledger/bee-rfcs/pull/0000)
++ Bee issue: [iotaledger/bee#0000](https://github.com/iotaledger/bee/issues/0000)
```

Comment:
You can fill this section in now. I added the corresponding tracking issue: [iotaledger/bee#80](https://github.com/iotaledger/bee/issues/80)

Reply:
Also don't forget to rename the file!